package.json

```diff
@@ -1,11 +1,17 @@
 {
-  "name": "d2-charts-api",
-  "version": "33.0.1",
+  "name": "@dhis2/d2-charts-api",
```

Comment:
@janhenrikoverland do you want it to be named `@dhis2/d2-charts-api` or just `@dhis2/charts-api`?

What's the plan for the package in the future?

Reply:
I would argue that since the lib does not depend on the backend, that it should not have `d2` in it. In addition, I'm not sure why it has `-api`. When the team first started talking about this lib, I assumed that it had something to do with the dhis2 rest api.

Reply:
I'm also wondering if we would actually want to move this to our new `d2-ui-analytics` package. @janhenrikoverland @edoardo @amcgee ?